### ISSUE TITLE
Villager regular trade missing

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#245]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/240

**Originally posted by InZanityINCORP (Jul 10, 2014):** 

What steps will reproduce the problem?
1. Enable the plugin with default config
2. Change the config/disable the pluginWhat is the expected output? What do you see instead?
After changing "disable-other-villagers" to false i still cant get access to the regular villager trade, neither does disabling the plugin entirely (Yes server has been restarted after each change)What version of the product are you using?
1.17.2

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (Jul 11, 2014):** 

I cannot reproduce this.

I started the server with normal villagers being disabled via the config of Shopkeepers, I then edited the config to turn that off (even while the server was running) and then reloaded the config via the command /shopkeepers reload. The villager I was testing with was offering it's trades like normal.

So make sure you have saved your config file (and properly uploaded those changes to your server as well). Maybe try stopping the server, then modifying the config and then then freshly start the server again. This sometimes seems to solve problems with config changes somehow not being properly applied if your are using some sort of special minecraft hosting provider.

            Edited Jul 11, 2014
